pkg/storage/replica_command.go

```diff
@@ -3073,8 +3073,24 @@ func splitTrigger(
 		rightLease := leftLease
 		rightLease.Replica = replica
 
+		gcThreshold, err := makeReplicaStateLoader(rec.RangeID()).loadGCThreshold(ctx, rec.Engine())
+		if err != nil {
+			return enginepb.MVCCStats{}, EvalResult{}, errors.Wrap(err, "unable to load GCThreshold")
+		}
+		if (gcThreshold == hlc.Timestamp{}) {
```

Comment:
When the cluster is young and GC has never run, I think we should actually see these happen, so I don't think we should emit a warning. Perhaps make it a `log.VEventf(ctx, 1, ...)`. Same for `txnSpanGCThreshold` below.